translations/ethos/de.csv

```diff
@@ -338,96 +338,96 @@ FAILSAFE_IS_NOT_SET,,"Failsafe ist nicht eingestellt"
+ADC_INSPECTOR,,"ADC-Wert-Prüfer"
 SYS_LONG,,"SYS lang"
 HOME_KEYMAP,,"Startseite Tastaturbelegung"
-POTS_SLIDERS_SETTINGS,,"Einst. Potis/Sliders"
+POTS_SLIDERS_SETTINGS,,"Einstellungen Potis/Slider"
 SWITCH_MIDDLE_DETECT_DELAY,,"Schaltermitte erkennt Verz."
 DISP_SHORT,,"{DISP_LABEL} kurz"
 DISP_LONG,,"{DISP_LABEL} lang"
 SWITCH,,"Schalter"
 ANALOGS_FILTER,,"Analoge Filter"
 ANALOGS_CALIBRATION,,"Kalib. analoge Geber","Kalib. anal. Geber"
-CONFIRM_CONTINUE,,"fortsetzen?"
-THR_NOT_LOW_WARNING,,"Gas nicht in Leerlaufstellung!"
-CHANNELS_ORDER,,"Kanal Reihenfolge"
-STICKS_MODE,,"Steuerknüppel Modus"
-STICKS_MODE_1,,"Steuerknüppel Modus 1"
-STICKS_MODE_2,,"Steuerknüppel Modus 2"
-STICKS_MODE_3,,"Steuerknüppel Modus 3"
-STICKS_MODE_4,,"Steuerknüppel Modus 4"
-FIRST_FOUR_CHANNELS_FIXED,,"Ersten 4 Kanäle sind festgel."
-NO_SD_CARD,,"Keine SD Karte"
-FLASH_BOOTLOADER,,"Flash Bootloader"
-FLASH_INTERNAL_MODULE,,"Flash Internes HF-Modul"
+CONFIRM_CONTINUE,,"Fortsetzen?"
+THR_NOT_LOW_WARNING,,"Gas ist nicht in Leerlaufstellung!"
+CHANNELS_ORDER,,"Kanal-Reihenfolge"
+STICKS_MODE,,"Steuerknüppel-Modus"
+STICKS_MODE_1,,"Steuerknüppel-Modus 1"
+STICKS_MODE_2,,"Steuerknüppel-Modus 2"
+STICKS_MODE_3,,"Steuerknüppel-Modus 3"
+STICKS_MODE_4,,"Steuerknüppel-Modus 4"
+FIRST_FOUR_CHANNELS_FIXED,,"Erste 4 Kanäle sind festgel."
+NO_SD_CARD,,"Keine SD-Karte"
+FLASH_BOOTLOADER,,"Bootloader flashen"
+FLASH_INTERNAL_MODULE,,"Internes HF-Modul flashen","Int. HF-Modul flashen"
 PLAY,,"abspielen"
-CURRENT_VERSION,,"Empf. Software Version"
-FLASH_RECEIVER_BY_INTERNAL_MODULE_OTA,,"Flash Empf. via int. OTA"
-FLASH_RECEIVER_BY_EXTERNAL_MODULE_OTA,,"Flash Empf. via ext. OTA"
-OTA_UPDATE,,"OTA Update"
-FLASH_EXTERNAL_DEVICE,,"Flash externe Geräte"
+CURRENT_VERSION,,"Firmwareversion Empf."
+FLASH_RECEIVER_BY_INTERNAL_MODULE_OTA,,"Empf. via int. OTA flashen"
+FLASH_RECEIVER_BY_EXTERNAL_MODULE_OTA,,"Empf. via ext. OTA flashen"
+OTA_UPDATE,,"OTA-Update"
+FLASH_EXTERNAL_DEVICE,,"Externe Geräte flashen"
 OPEN,,"öffnen"
-FLASH_EXTERNAL_MODULE,,"Flash Externes Modul"
-FLASH_EXTERNAL_MULTIMODULE,,"Flashe externes Multi-Modul"
-MOVE_HERE,,"versch. hierher"
-OTA_UPDATE_ERROR,,"OTA Update Fehler"
+FLASH_EXTERNAL_MODULE,,"Externes HF-Modul flashen","Ext. HF-Modul flashen"
+FLASH_EXTERNAL_MULTIMODULE,,"Externes Multi-Modul flashen","Ext. Multi-Modul flashen"
+MOVE_HERE,,"Hierher versch."
+OTA_UPDATE_ERROR,,"OTA-Update Fehler"
 UNSUPPORTED_RX,,"Empf. nicht unterstützt"
-MAIN_VOLTAGE,,"Senderakku -kalibriert?-"
+MAIN_VOLTAGE,,"Sender-Batteriespannung"
 DISPLAY_VOLTAGE_RANGE,,"Anzeige Spannungsbereich"
-LOW_VOLTAGE,,"Warnschwelle Akkuspannung"
-RTC_VOLTAGE,,"Uhr/Datum -Batterie-"
+LOW_VOLTAGE,,"Warnschwelle Batteriespannung"
+RTC_VOLTAGE,,"RTC-Batterie"
 UNKNOWN,,"unbekannt"
 FIRMWARE,,"Firmware"
-FIRMWARE_VERSION,,"Version (Firmware)"
-VIDEO_HARDWARE_VERSION,,"Video Hardware Version"
-RECEIVER,,"Empfänger"
-NON_STABILIZED_RECEIVER,,"Kein stabi. Empf."
+FIRMWARE_VERSION,,"Firmwareversion"
```

Comment:
version is lowercase here, while it is capitalized in the next line, on purpose?